Mesh_3/include/CGAL/Mesh_3/search_for_connected_components_in_labeled_image.h

```diff
@@ -137,7 +137,9 @@ search_for_connected_components_in_labeled_image(const CGAL::Image_3& image,
             {
               visited[offset] = true;
               second_pass[offset] = false;
+#if CGAL_MESH_3_SEARCH_FOR_CONNECTED_COMPONENTS_IN_LABELED_IMAGE_VERBOSE > 1
               ++nb_voxels;
+#endif
```

Comment:
This is not correct. The `nb_voxels` variable is used here:
https://github.com/afabri/cgal/blob/8674744f1719ced37df9ef566ea709ec603b9a42/Mesh_3/include/CGAL/Mesh_3/search_for_connected_components_in_labeled_image.h#L228-L239

```suggestion
#ifdef CGAL_MESH_3_SEARCH_FOR_CONNECTED_COMPONENTS_IN_LABELED_IMAGE_VERBOSE
              ++nb_voxels;
#endif
```